lib/ex_webrtc/rtp/h264.ex

```diff
@@ -0,0 +1,53 @@
+defmodule ExWebRTC.RTP.H264 do
+  @moduledoc """
+  Utilities for RTP packets carrying H264 encoded payload.
+  """
+
+  alias ExRTP.Packet
+
+  # Copied nearly 1-to-1 from https://github.com/membraneframework/membrane_rtp_h264_plugin/blob/master/lib/rtp_h264/utils.ex
```

Comment:
Can we please copy relevant comment from Membrane RTP plugin to make this chain a litte shorter? Just mention that we took the code from galene and it is unclear why we check against nalu type 5 and 7 instead of just one of them